SimpleWAWS/Code/BackgroundQueueManager.cs

```diff
@@ -199,7 +204,7 @@ private void DeleteAndCreateResourceGroupOperation(ResourceGroup resourceGroup)
             {
                 Description = $"Deleting and creating resourceGroup {resourceGroup.ResourceGroupName}",
                 Type = OperationType.ResourceGroupDeleteThenCreate,
-                Task = resourceGroup.DeleteAndCreateReplacement(),
+                Task = resourceGroup.DeleteAndCreateReplacement(false),
```

Comment:
use named parameters when passing a boolean to a function. For example `DeleteAndCreateReplacement(log: false)`  or whatever the flag is actually for